packages/gatsby/src/utils/__tests__/__snapshots__/handle-flags.ts.snap

```diff
@@ -50,11 +50,9 @@ The following flags were automatically enabled on your site:
 - PARTIAL_RELEASE · (Umbrella Issue (test)) · test
 - PARTIAL_RELEASE_ONLY_NEW_LODASH · (Umbrella Issue (test)) · test
 
-There are 5 other flags available that you might be interested in:
+There are 3 other flags available that you might be interested in:
 - FAST_DEV · Enable all experiments aimed at improving develop server start time
 - DEV_SSR · (Umbrella Issue (https://github.com/gatsbyjs/gatsby/discussions/28138)) · SSR pages on full reloads during develop. Helps you detect SSR bugs and fix them without needing to do full builds.
-- QUERY_ON_DEMAND · (Umbrella Issue (https://github.com/gatsbyjs/gatsby/discussions/27620)) · Only run queries when needed instead of running all queries upfront. Speeds starting the develop server.
```

Comment:
this is removed because `QUERY_ON_DEMAND` is not available in CI (side note - because it's locked in we probably shouldn't display it for other reasons, but that seems separate)